openacademy/models/openacademy_course.py

```diff
@@ -8,5 +8,35 @@ class Course(models.Model):
 
     name = fields.Char(string="Title", required=True)
     description = fields.Text()
+    responsible_id = fields.Many2one(
+            'res.users', string="Responsible",
+            index=True, ondelete='set null', default=lambda self, *a: self.env.uid)
+    session_ids = fields.One2many('openacademy.session', 'course_id')
+
+    _sql_constraints = [
+            ('name_description_check',
+            'CHECK( name != description  )',
+            "The title of the course should not be the description"
+            ),
+            ('name_unique',
```

Comment:
these lines fits in a single line

Reply:
This change is missing @jjbenavaz 

@hugho-ad means put all this in one line

```suggestion
            ('name_unique', 'UNIQUE(name)', "The course title must be unique"),
```